x-pack/legacy/plugins/infra/public/components/source_configuration/indices_configuration_panel.tsx

```diff
@@ -41,97 +43,101 @@ export const IndicesConfigurationPanel = ({
       </h3>
     </EuiTitle>
     <EuiSpacer size="m" />
-    <EuiDescribedFormGroup
-      idAria="matricIndices"
-      title={
-        <FormattedMessage
-          id="xpack.infra.sourceConfiguration.metricIndicesTitle"
-          defaultMessage="Metric indices"
-        />
-      }
-      description={
-        <FormattedMessage
-          id="xpack.infra.sourceConfiguration.metricIndicesDescription"
-          defaultMessage="Index pattern for matching indices that contain Metricbeat data"
-        />
-      }
-    >
-      <EuiFormRow
-        describedByIds={['metricIndices']}
-        error={metricAliasFieldProps.error}
-        fullWidth
-        helpText={
+    {displaySettings === 'metrics' && (
+      <EuiDescribedFormGroup
+        idAria="matricIndices"
```

Comment:
This was incorrect before, but maybe we can use this opportunity to fix it:

```suggestion
        idAria="metricIndices"
```